src/main/java/org/verapdf/pd/font/cff/CFFFileBaseParser.java

```diff
@@ -55,6 +55,7 @@ protected int readCard16() throws IOException {
     }
 
     private long readOffset(int offSize) throws IOException {
+        assert offSize >= 1;
```

Comment:
Could you remove or replace this with throwing IOException

Reply:
Sure! My thinking was that since it´s a private method, any offSize less than 1 is an indication of a flaw internal to CFFFileBaseParser, and therefore `assert` would do, but I´ll change it to throw an `IOException`.

Reply:
Changed in https://github.com/veraPDF/veraPDF-parser/compare/29d798ccc0ef6d42df8732aecc0dbfed99798b6e..ba542e7c88b799a514cf816d49beb2d1d1b702a4